### PR TITLE
alert and button added

### PR DESCRIPTION
**Pull request by jeetbhalani15**
alert and buttons components

### CODE REVIEW

File: components/buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    padding: 0.5rem 1rem;
+    margin: 1.25rem 0;
+    /* border-radius: 0.3rem; */
+    background-color: transparent;
+    /* border: none; */
+    border: 2px solid grey;
+    font-size: 1rem;
+    text-align: center;
+    width: 12rem;
+    /* outline: none; */
```
Comment (anupkgurung) [inline components/buttons/button.css]: remove unwanted code instead of cementing

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+.close-btn{
+    font-weight: 600;
+    background: none;
+    border: none;
+    position: absolute;
+    right: 1%;
+    /* top: 36%; */
+    bottom: 36%;
+}
```
Comment (Rohitprasad83) [inline components/alert/alert.css:71]: Since color hex codes are repeated, please use variables for them

File: components/buttons/button.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    cursor: pointer;
+}
+.btn-solid{
+    background-color: var(--btn-color);
+}
+.btn-outline{
+    border: 2px solid #0dafa0;
+    color: #159673;
```
Comment (anupkgurung) [inline components/buttons/button.css]: move colors to a separate file where all common properties will reside

File: components/buttons/button.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+*/
+.btn-float{
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 50px;
+    height: 50px;
+    font-size: 1rem;
```
Comment (anupkgurung) [inline components/buttons/button.css:98]: instead of px try to use rem. if values are not greater than 5px or 10px

**Review by Rohitprasad83 — CHANGES_REQUESTED** [review]
more css color variables could be used. Also try to provide a gif or picture of your working component.

**Review by anupkgurung — COMMENTED** [review]
rest seems to be fine. Please refer to comments.

File: components/alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    color: #b16501;
+    background-color: var(--warning-color);
+}
+/* 
+CSS FOR OULINE ALERT
+*/  
+.alert-primary-outline{
+    background-color: #ffffff;
+    color: #1c316b;
+    border: 2px solid blue;
```
Comment (Naveenchandar) [inline components/alert/alert.css:42]: Please use hex codes for consistency

File: components/alert/alert.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+}
+.alert-danger-outline{
+    color: #ee2626;
+    background-color: #ffffff;
```
Comment (Naveenchandar) [inline components/alert/alert.css]: Background-color property is repeated. Try to use a descendant selectors or use common class name.

File: components/alert/alert.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+const alertBox= document.querySelector('.alert-btn');
+const btn = document.querySelector('.close-btn');
+
+btn.addEventListener('click', ()=>{
+    alertBox.style.display="none"
+})
+console.log("hello")
```
Comment (Naveenchandar) [inline components/alert/alert.js]: You can remove the console.

File: components/buttons/button.css [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+    content: "";
+    top:0;
+    left: 9px;
+    width: 26px;
+    height: 26px;
+    border-radius: 50%;
+    background-color: rgb(226, 226, 226);
+    transition: 0.4s;
+}
+.btn-toggle .checkbox:checked + .btn-slider::before{
```
Comment (Naveenchandar) [inline components/buttons/button.css:81]: Try to avaoid multiple nested classes. Single nesting is fine.

File: components/alert/alert.css (hunk 2 above)
Comment (jeetbhalani15) [reply components/alert/alert.css:71]: ok done

File: components/buttons/button.css (hunk 4 above)
Comment (jeetbhalani15) [reply components/buttons/button.css:98]: ok done

File: components/buttons/button.css (hunk 8 above)
Comment (jeetbhalani15) [reply components/buttons/button.css:81]: sure i will try to avoid this mistakes

File: components/buttons/button.css (hunk 3 above)
Comment (jeetbhalani15) [reply components/buttons/button.css]: ok done

**Pull request merged**